src/store/store.js

```diff
@@ -1,13 +1,20 @@
-import { createStore, combineReducers } from 'redux';
+import { createStore, combineReducers, applyMiddleware } from 'redux';
 import { composeWithDevTools } from 'redux-devtools-extension';
+import thunkMiddleware from 'redux-thunk';
 
-import loginReducer from './login/reducer';
+import loginReducer from './login/reducers';
+import registerReducer from './register/reducers';
 
 const store = createStore(
-  combineReducers({
-    login: loginReducer,
-  }),
-  composeWithDevTools()
+  combineReducers(
+    {
+      login: loginReducer,
+      register: registerReducer,
+    },
+    composeWithDevTools()
```

Comment:
i actually don't think you need this line here. A mistake i made writting the sheets